tagcwebapp/src/main/resources/index.html

```diff
@@ -6,7 +6,7 @@
 Your browser does not support the HTML5 canvas tag.</canvas>
 	<button type="button" id="zoominbtn">Zoom in</button>
 	<button type="button" id="zoomoutbtn">Zoom out</button>
-<script src="//code.jquery.com/jquery-1.11.0.min.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
```

Comment:
This should be a local jquery file, to prevent the system from not working without internet.